csharp_dotnetcore/9.Message-Routing/CognitiveModels/LUIS/General.cs

```diff
@@ -0,0 +1,81 @@
+// <auto-generated>
+// Code generated by LUISGen
+// Tool github: https://github.com/microsoft/botbuilder-tools
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.AI.Luis;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Luis
+{
+    public class MessageRoutingBot_General : IRecognizerConvert
+    {
+        public string Text;
+        public string AlteredText;
+        public enum Intent
+        {
+            Greeting,
+            Help,
+            Cancel,
+            Restart,
+            Escalate,
+            ConfirmYes,
+            ConfirmNo,
+            ConfirmMore,
+            Next,
+            Goodbye,
+            None
+        };
+        public Dictionary<Intent, IntentScore> Intents;
+
+        public class _Entities
+        {
```

Comment:
Lots of interesting things here. No comments, 

Also, what's with the "_" naming convention?

Reply:
@cleemullins this is generated by luisgen, as stated in the upper part. Yes, it does not match our patterns :(